packages/create-mdx/readme.md

```diff
@@ -37,9 +37,9 @@ abide by its terms.
 
 <!-- Definitions -->
 
-[build]: https://travis-ci.org/mdx-js/mdx
+[build]: https://travis-ci.com/mdx-js/mdx
```

Comment:
Is .com correct? 🤔 [org](https://api.travis-ci.org/mdx-js/mdx.svg?branch=master) works for me, [com](https://travis-ci.com/mdx-js/mdx.svg?branch=master) doesn't?

Reply:
Working to get the repo migrated over to the updated Travis CI https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps

Reply:
https://docs.travis-ci.com/user/migrate/open-source-repository-migration

Reply:
It took a couple toggles to get it right, but the migration appears to have stuck this time.

Reply:
Confirmed, https://github.com/mdx-js/mdx/pull/599/checks is now populated. Which only the new Travis CI integration does.

Reply:
wow that’s cool!